var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -39,11 +39,21 @@ class Icu4c(AutotoolsPackage):
     version('58.2', 'fac212b32b7ec7ab007a12dff1f3aea1')
     version('57.1', '976734806026a4ef8bdd17937c8898b9')
 
+    # The --enable-rpath option is needed on MacOS, but it breaks the
+    # build for xerces-c on Linux.
+    variant('rpath', default=True,
```

Comment:
can you add the breaks-linux problem in the description? Also please add a `conflicts(~rpath, when=platform=Darwin)` and the inverse for linux